db_schema/schema.sql

```diff
@@ -1,23 +1,20 @@
-DROP TABLE IF EXISTS events;
-CREATE TABLE events (
+CREATE TABLE IF NOT EXISTS events (
```

Comment:
Good idea to change `DROP TABLE; CREATE TABLE` TO `CREATE TABLE IF NOT EXISTS`. Shall we do the same change in the other repos, too? Or will we remove the schemas from the other repos eventually?

Reply:
Nevermind. Just saw your [other PR in API](https://github.com/openpodcast/stack/pull/3). 😬